### PR TITLE
Update Name Request Front Page #8715

### PR DESCRIPTION
**Pull request by eve-git**
*Issue #:* /bcgov/entity#8715

*Description of changes:*

In the name request page:
1. put the list item (options) of a dropdown under the select input, instead of on the select input
2. change the color of an unselected list item to #495057
3. change the Jurisdiction list item text of 'BC' to 'British Columbia'

In edit name page:
1. adjust column width

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/components/common/names-capture.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
         </v-row>
 
         <v-row class="mt-3">
-          <v-col cols="5" class="py-0">
+        <v-col cols="5" class="py-0">
```
Comment (cameron-freshworks) [inline src/components/common/names-capture.vue:11]: i believe you still want an indent here as the `v-col` is a child element of the `v-row`.

File: src/components/new-request/search.vue [hunk 2]
```diff
@@ -566,5 +572,8 @@ export default class NewSearch extends Mixins(CommonMixin) {
   .v-messages:not(.error--text) {
     margin-bottom: -22px;
   }
+  .theme--light.v-list-item:not(.v-list-item--active):not(.v-list-item--disabled) {
+    color: #495057 !important;
```
Comment (cameron-freshworks) [inline src/components/new-request/search.vue:576]: You can use a predefined theme colour opposed to the hex here, i believe this is `$gray7`.

**Review by cameron-freshworks — COMMENTED** [review]
Looks good! Just a few minor things :)

File: src/components/new-request/search.vue [hunk 3]
```diff
@@ -50,7 +51,8 @@
                         :disabled="locationDisabled"
                         :readonly="!request_action_cd"
                         :class="!request_action_cd ? 'disabled-custom' : ''"
-                        @change="clearErrors()"
+                        :menu-props="{ bottom: true, offsetY: true}"
+                       @change="clearErrors()"
```
Comment (andrepestana-aot) [inline src/components/new-request/search.vue:55]: There's an issue with the indentation here as well.

**Review by andrepestana-aot — CHANGES_REQUESTED** [review]

**Comment by eve-git** [comment]
@cameron-freshworks @andrepestana-aot please review again. Thanks!

**Review by andrepestana-aot — APPROVED** [review]
Looks good!

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

**Pull request merged**